litmus/libdir/_aarch64/kvm_symb.c

```diff
@@ -0,0 +1,26 @@
+
+/* Generic type of symbolic value generated from the final state of a litmus
+ * test or the FAR_EL1 register in case of a fault.
+ */
+typedef struct {
+  uintptr_t offset; // offset between the raw value and the symbolic value
+  int id; // id of the symbolic value
+} symb_t;
+
+static symb_t symbolic_of_id(int id) {
+  symb_t ret = { .id = id, .offset = 0 };
+  return ret;
+}
+
+static int symbolic_equal(symb_t s1, symb_t s2) {
+  return s1.id == s2.id && s1.offset == s2.offset;
+}
```

Comment:
In some occasions those functions may be included and not called, which will yield a warning. As the functions are small, an easy workaround can be to declare them as `inline static`.